cub/device/dispatch/tuning/tuning_scan.cuh

```diff
@@ -0,0 +1,205 @@
+          accum_size AccumSize                 = classify_accum_size<AccumT>()>
+struct sm90_tuning
+{
+  static constexpr int threads = 128;
+  static constexpr int items   = 15;
+
+  using delay_constructor = detail::default_delay_constructor_t<AccumT>;
+};
+
+// clang-format off
+template <class T> struct sm90_tuning<T, primitive_op::yes, primitive_accum::yes, accum_size::_1> : tuning<192, 22, 168, 1140> {};
+template <class T> struct sm90_tuning<T, primitive_op::yes, primitive_accum::yes, accum_size::_2> : tuning<512, 12, 376, 1125> {};
+template <class T> struct sm90_tuning<T, primitive_op::yes, primitive_accum::yes, accum_size::_4> : tuning<128, 24, 648, 1245> {};
+template <class T> struct sm90_tuning<T, primitive_op::yes, primitive_accum::yes, accum_size::_8> : tuning<224, 24, 632, 1290> {};
+
+template <> struct sm90_tuning<float,  primitive_op::yes, primitive_accum::yes, accum_size::_4> : tuning<128, 24, 688, 1140> {};
+template <> struct sm90_tuning<double, primitive_op::yes, primitive_accum::yes, accum_size::_8> : tuning<224, 24, 576, 1215> {};
+
+#if CUB_IS_INT128_ENABLED 
+template <> struct sm90_tuning< __int128_t, primitive_op::yes, primitive_accum::no, accum_size::_16> : tuning<576, 21, 860, 630> {};
+template <> struct sm90_tuning<__uint128_t, primitive_op::yes, primitive_accum::no, accum_size::_16> : tuning<576, 21, 860, 630> {};
+#endif
+// clang-format on
+
+} // namespace scan
+} // namespace detail
+
+
+template <typename AccumT, typename ScanOpT = Sum> 
+struct DeviceScanPolicy
+{
+  // For large values, use timesliced loads/stores to fit shared memory.
+  static constexpr bool LargeValues = sizeof(AccumT) > 128;
+  static constexpr BlockLoadAlgorithm ScanTransposedLoad =
+    LargeValues ? BLOCK_LOAD_WARP_TRANSPOSE_TIMESLICED
+                : BLOCK_LOAD_WARP_TRANSPOSE;
+  static constexpr BlockStoreAlgorithm ScanTransposedStore =
+    LargeValues ? BLOCK_STORE_WARP_TRANSPOSE_TIMESLICED
+                : BLOCK_STORE_WARP_TRANSPOSE;
+
+  template <int NOMINAL_BLOCK_THREADS_4B,
+            int NOMINAL_ITEMS_PER_THREAD_4B,
+            typename ComputeT,
+            BlockLoadAlgorithm LOAD_ALGORITHM,
+            CacheLoadModifier LOAD_MODIFIER,
+            BlockStoreAlgorithm STORE_ALGORITHM,
+            BlockScanAlgorithm SCAN_ALGORITHM,
+            typename DelayConstructorT>
+  using policy_t =
+    AgentScanPolicy<NOMINAL_BLOCK_THREADS_4B,
+                    NOMINAL_ITEMS_PER_THREAD_4B,
+                    ComputeT,
+                    LOAD_ALGORITHM,
+                    LOAD_MODIFIER,
+                    STORE_ALGORITHM,
+                    SCAN_ALGORITHM,
+                    MemBoundScaling<NOMINAL_BLOCK_THREADS_4B, NOMINAL_ITEMS_PER_THREAD_4B, ComputeT>,
+                    DelayConstructorT>;
+
+  /// SM350
+  struct Policy350 : ChainedPolicy<350, Policy350, Policy350>
+  {
+    // GTX Titan: 29.5B items/s (232.4 GB/s) @ 48M 32-bit T
```

Comment:
Do we want to keep those stats?

If so do we want to add them for all policies?

Reply:
We might at some point, but I don't want to invest a lot of time into this.